draniksscar/Elder_Priest_of_Discord.lua

```diff
@@ -0,0 +1,129 @@
+            e.other:MovePC(Zone.grobb, 2, 13, 4, 180)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == Zone.erudnext then
+            e.self:Say("Very well.  You are now to return from whence you came.")
+            e.other:DelGlobal("OOW_PoD_Origin")
+            e.other:MovePC(Zone.erudnext, 30, -130, 40, 180)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == Zone.neriakb then
+            e.self:Say("Very well.  You are now to return from whence you came.")
+            e.other:DelGlobal("OOW_PoD_Origin")
+            e.other:MovePC(Zone.neriakb, -564, -43, -37, 192)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == Zone.freporte then
+            e.self:Say("Very well.  You are now to return from whence you came.")
+            e.other:DelGlobal("OOW_PoD_Origin")
+            if eq.is_prophecy_of_ro_enabled() then
+                e.other:MovePC(Zone.freeporteast, -682, 146, -51, 192)
+                return
+            end
+            e.other:MovePC(Zone.freporte, -682, 146, -51, 192)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == Zone.felwithea then
+            e.self:Say("Very well.  You are now to return from whence you came.")
+            e.other:DelGlobal("OOW_PoD_Origin")
+            e.other:MovePC(Zone.felwithea, -289, 10, 4, 192)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == Zone.gfaydark then
+            e.self:Say("Very well.  You are now to return from whence you came.")
+            e.other:DelGlobal("OOW_PoD_Origin")
+            e.other:MovePC(Zone.gfaydark, 215, 232, 2, 192)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == Zone.halas then
+            e.self:Say("Very well.  You are now to return from whence you came.")
+            e.other:DelGlobal("OOW_PoD_Origin")
+            e.other:MovePC(Zone.halas, 47, 123, 4, 10)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == Zone.oggok then
+            e.self:Say("Very well.  You are now to return from whence you came.")
+            e.other:DelGlobal("OOW_PoD_Origin")
+            e.other:MovePC(Zone.oggok, 500, 220, 60, 10)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == Zone.qeynos2 then
+            e.self:Say("Very well.  You are now to return from whence you came.")
+            e.other:DelGlobal("OOW_PoD_Origin")
+            e.other:MovePC(Zone.qeynos2, 227, 22, 3, 10)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == Zone.rivervale then
+            e.self:Say("Very well.  You are now to return from whence you came.")
+            e.other:DelGlobal("OOW_PoD_Origin")
+            e.other:MovePC(Zone.rivervale, -110, -16, 4, 192)
+            return
+        end
+        if e.other:GetGlobal("OOW_PoD_Origin") == 55 then
```

Comment:
`55` should be a constant, right?

Reply:
Correct. Also seems trust has a tool to turn this into a global script, with his PoR effort

Reply:
Added. This will be in draft for a while, there's a bunch of coords needing updates we're working on

Reply:
Bleh, looks like formatter applied too. I'll reverse it.